code/dynamic_programming/src/Array_Median/median.cpp

```diff
@@ -0,0 +1,25 @@
+#include<iostream>
+#include<algorithm>
+using namespace std;
+
+float median(int n, int arr[n])
+{
+	if (n % 2 != 0)
+		return (float)arr[n / 2];
+
+	return ((float)(arr[(n - 1) / 2] + arr[n / 2]) / 2);
+}
+
+int main()
+{
+	int n;
+	cout<<"Enter size of the Array";
+	cin>>n;
+	int arr[n];
+	cout<<"Enter "<<n<<" integers";
+	for (int i = 0; i < n; i++)
+		cin>>arr[i];
+	sort(arr,arr+n);
+	cout<<"Median= "<<median(n,arr);
```

Comment:
Add a space after commas and between operators (https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style/c%2B%2B#spaces).